src/app/action-template/action-template.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component, OnInit } from '@angular/core';
+
+
+const ACTIONS: string[] = ['Addition','Deletion','Update'];
```

Comment:
outside variable is not cool, put in a static class variable if you need to have it.